docs/get_started/getting_started_riscv_cmake.md

```diff
@@ -35,21 +29,37 @@ $ ./build_tools/riscv/riscv_bootstrap.sh
 
 ## Configure and build
 
-### Configure on Linux
+### Host configuration
+
+Build and install at least the compiler tools on your host machine, or install them from a binary distribution:
+
+```shell
+$ mkdir build-host ; cd build-host ; mkdir install
+$ cmake -G Ninja -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_INSTALL_PREFIX=./install ..
+$ cmake --build . --target install
```

Comment:
nit: I notice for Android, the host compiler is built outside of the iree root directory. Maybe for the sake of consistency, the shell script here can be the same? The host compiler should be shareable between Android and RiscV targets.

Reply:
+1, use `../iree-build-host/` as in the Android docs: https://google.github.io/iree/get-started/getting-started-android-cmake
```
$ cmake -G Ninja -B ../iree-build-host/ -DCMAKE_INSTALL_PREFIX=../iree-build-host/install .
$ cmake --build ../iree-build-host/ --target install
```

Directory structure is flexible, but we recommend putting build artifacts out of tree

recommended:
```
/dev/projects/iree/
/dev/projects/iree-build-host/ 
/dev/projects/iree-build-android/
```

should work, but not recommended by our docs:
```
/dev/projects/iree/
/dev/projects/iree/build-host/ 
/dev/projects/iree/build-android/
```

Reply:
Got it.